storage/EtcFstab.cc

```diff
@@ -286,8 +290,12 @@ namespace storage
             fs_type = FsType::UNKNOWN;
 
 	mount_opts.parse( EtcFstab::fstab_decode( get_column( col++ ) ), line_no );
-	dump_pass = atoi( get_column( col++ ).c_str() );
-	fsck_pass = atoi( get_column( col++ ).c_str() );
+
+	if ( column_count >= FSTAB_COLUMN_COUNT - 1 )
+	    dump_pass = atoi( get_column( col++ ).c_str() );
```

Comment:
```dump_pass``` should be set to ```0``` if the value is not present. The ```parse``` function could be used on an entry that was not just created with clean values.